### DISCUSSION TITLE
problem with local fonts in nextjs 13

### DISCUSSION THREAD

**Discussion by mohammadshariat** [vercel/next.js#43431] [Help]
Hi ... I have problem with local fonts in nextjs 13 in after bulild the project in production ... before nextjs 13 I added fonts in style.css in styles folder like this :

` @font-face {
      font-family: Morabba;
      font-style: normal;
      font-weight: 500;
      src: url("/fonts/Morabba/woff/Morabba-medium.woff") format("woff"),
         url("/fonts/Morabba/woff2/Morabba-medium.woff2") format("woff2");
   }

   @font-face {
      font-family: Morabba;
      font-style: normal;
      font-weight: 600;
      src: url("/fonts/Morabba/woff/Morabba-semibold.woff") format("woff"),
         url("/fonts/Morabba/woff2/Morabba-semibold.woff2") format("woff2");
   }

   @font-face {
      font-family: Morabba;
      font-style: normal;
      font-weight: bold;
      src: url("/fonts/Morabba/woff/Morabba-bold.woff") format("woff"),
         url("/fonts/Morabba/woff2/Morabba-bold.woff2") format("woff2");
   }`

but now this isnt work in production and i have this error 
`Failed to load resource: the server responded with a status of 404 (Not Found)`

i use (tailwindcss) in next js , if it helps

thanks

**Comment by ephraimduncan** [comment]
Use the `@next/font` package that comes with Next.js 13

**Reply by ghost** [reply]
+1
Also using tailwind. I've tried declaring fonts in `layout.tsx` using `variable` to no effect either.
